frontend/src/pages/profile/components/user-info/user-info.tsx

```diff
@@ -12,39 +12,44 @@ type Properties = {
 };
 
 const UserInfo: React.FC<Properties> = ({ user, className }) => {
-  const [isEditingProfile, setIsEditingProfile] = useState(false);
+  const { isOpen, handleToggleModalOpen } = useModal();
   const userName = getFullName(user.firstName, user.lastName);
 
-  const handleEditMode = useCallback(
-    (value = true) => setIsEditingProfile(value),
-    [],
-  );
+  const handleModalClose = useCallback(() => {
+    if (isOpen) {
+      handleToggleModalOpen();
+    }
+    return false;
```

Comment:
Do we need `return false` here?

Reply:
😁
I just copied this handler from another place.

Reply:
[037e88f](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/395/commits/037e88f8f7a3eda98fb9e9edd912733037ff44f0)